modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,24 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you make a {pulsar-x} shot:
+
+IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but it still only counts as a single super-heavy weapon.
+The resultant salvo of {macro-weapon} shots does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
+
+. Determine the number of shots in the salvo, based on the value of `(X)` that we specify -- for example, _`(D3+1)`_ or _`(D6)`_.
+footnote:[{e40k-diff}
+In {e40k}, the 'salvo' value of {pulsar} is D6 only.
```

Comment:
```suggestion
In {e40k}, the 'salvo' value of {pulsar} is D6 only.
```

One of the reasons for updating the Pulsar is that it wasn't clear if it _was_ always D6 in Epic 40k. The Revenant's 'Pulse Laser' was formatted in exactly the same way as the Phantom's Pulsar, but with lower range and Firepower D3 instead of Firepower D6.

Reply:
Hmm. I see your point, but I was simply trying to highlight the fact that the original _Pulsar_ rules text is explicitly D6 and nothing else, and we've made it more flexible.

Yes, there is the issue that it seems like the Pulse Laser on the Revenant Titan should work in the same way as the Pulsar, except with a different 'salvo' value, or else the Blast markers get silly — but I'm not sure we really need to spell all that out here, even if it is a major reason for the change.

_If we need to get into it at all, another option might be to mention it as another 'difference' note in close proximity to the Revenant Titan data sheet — the old one didn't mention the _Pulsar_ ability whereas the new one will state _Pulsar (D3)_ or whatever it ends up being._

If you think we **should** spell it out here then I'm not dead against it — feel free to suggest some text. You might want to resolve this thread and then suggest across multiple lines to integrate this specific point with the other more general points I made about balance, etcetera.